modules/wildfly/pom.xml

```diff
@@ -51,6 +51,13 @@
             <groupId>org.hibernate.ogm</groupId>
             <artifactId>hibernate-ogm-infinispan</artifactId>
             <optional>true</optional>
+            <exclusions>
+                <exclusion>
+                    <!-- The Lucene dependency is not compatible with Neo4j embedded -->
+                    <groupId>org.hibernate</groupId>
+                    <artifactId>hibernate-search-engine</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
The infinispan WildFly module is not using it (it uses the one included in WilfFly) there is really no reason to have it as dependency in this module. If I don't exclude it I cannot include the right version for Neo4j.

Also the integration tests will fail if for some reason a module is missing a library or it is using the wrong version.